### PR TITLE
Be/notifications

### PR DESCRIPTION
**Pull request by thawk55**
Notification CRUD, basic tests, and swagger docs.

### CODE REVIEW

File: api/media/media.controller.js [hunk 1]
```diff
@@ -50,8 +50,8 @@ exports.getMediaItem = function(req, res){
 exports.addMedia = function(req, res){
   //Add to database
   media.create(req.body, function(err, results) {
-    if(err) { 
-      return handleError(res, err); 
+    if(err) {
```
Comment (jeffreykbacon) [inline api/media/media.controller.js]: Are you running some sort of linter to find the extra spaces?  If so, what are you using?

File: api/media/media.controller.js [hunk 2]
```diff
@@ -39,8 +39,8 @@ exports.getMediaItem = function(req, res){
     if(!results){
       return res.send(404);
     }
-    
-    return res.json(results);
+
+    return res.json(results[0]);
```
Comment (jeffreykbacon) [inline api/media/media.controller.js]: Could we use mongoose's findOne method instead of grabbing the first item of the array?

**Comment by jeffreykbacon** [comment]
Looks good to me! :+1:

File: api/media/media.controller.js (hunk 1 above)
Comment (thawk55) [inline api/media/media.controller.js]: I think Atom just does it automatically. I don't recall changing that at all.

**Pull request merged**